cloud/migrate-to-cloud/hypershift.md

```diff
@@ -0,0 +1,172 @@
+excerpt: Migrate an existing PostgreSQL database to Timescale Cloud in a single step
+product: cloud
+keywords: [data migration, Hypershift]
+tags: [ingest, Hypershift, postgresql]
+---
+
+import Experimental from 'versionContent/_partials/_experimental.mdx';
+
+# Migrate with Hypershift
+
+You can use Hypershift&nbsp;0.2&nbsp;(beta) to migrate existing PostgreSQL
+databases in one step, and enable compression and hypertable creation on the
+fly.
+
+Hypershift can migrate your data in Timescale Cloud from these sources:
+
+*   Standard PostgreSQL databases
+*   Amazon RDS databases
+*   Other TimescaleDB databases, including Managed Service for Timescale
+
+Because compression is enabled during the migration, you do not need to have the
+maximum amount of storage available in the target database before you start
+migration.
+
+<Experimental />
+
+In preliminary testing, Hypershift migrated 60&nbsp;GB of data in 9 to 12 minutes,
+and 1&nbsp;TB of data in under 4 hours. You can continue reading from your
+source database during this time, though performance could be slower. If you
+write to tables in your source database during the migration, the new writes
+might not be transferred to Timescale Cloud. To avoid this problem, fork your
+database and migrate your data from the fork. To avoid this problem, see the
+section on [migrating an active database]
+(<http://docs.timescale.com/cloud/latest/migrate-to-cloud/#migrate-an-active-database>).
+
+<highlight type="important">
+If you have a very large database, and Hypershift is going to have to run for a
+very long time to migrate it, for example, a day or more, ensure that you have a
+very stable network connection. Hypershift is not able to recover if the network
+connection is interrupted.
+</highlight>
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+*   Signed up for your [free Timescale Cloud account][cloud-install].
+*   Installed [Docker][docker-install].
+
+<highlight type="cloud" header="Run all tutorials free" button="Try for free">
+Your Timescale Cloud trial is completely free for you to use for the first
+thirty days. This gives you enough time to complete all the tutorials and run
+a few test projects of your own.
+</highlight>
+
+## Download the Hypershift container
+
+<Tabs label="Download Hypershift">
+
+<Tab title="MacOS">
+
+<procedure>
+
+### Downloading the Hypershift container on MacOS
+
+1.  Open the Docker app on your local machine to start the service.
+
+1.  At the command prompt, pull the latest Hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
+    ```
+
+</procedure>
+
+</Tab>
+
+<Tab title="Linux">
+
+<procedure>
+
+### Downloading the Hypershift container on Linux
+
+1.  Start the Docker service:
+
+    ```bash
+    sudo service docker start
+    ```
+
+1.  At the command prompt, pull the latest Hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
```

Comment:
```suggestion
    docker pull timescale/hypershift:0.2
```

Reply:
Doesn't this mean that we would need to update the tag with each release? I thought if we left it as just `hypershift` then it will automatically pull the latest one?

Reply:
If the docker image has a `latest` tag, then that is the default behaviour, yes. Our image (currently) doesn't have a `latest` tag. I generally don't like proposing that people use the `latest` tag, because it invariably ends up in a script, which breaks when a new version is released. I understand the annoyance/difficulty here of keeping docs and code in sync though, so now I'm even more on the fence about whether we should have a `latest` tag or not..

Reply:
Happy to put the tag in for now. If you add the `latest` tag, though, let me know 🚀